### ISSUE TITLE
Unable to install `gval`

### ISSUE THREAD

**Issue by jarq6c** [NOAA-OWP/gval#104]
Python 3.10.10 (main, Feb  9 2023, 05:30:13) [GCC 10.2.1 20210110] on linux

I attempted to install `gval` using the following:
```bash
$ pip install 'gval @ https://github.com/NOAA-OWP/gval'
```

I got this output.
```console
Collecting gval@ https://github.com/NOAA-OWP/gval
  Downloading https://github.com/NOAA-OWP/gval
     \ 239.6 kB 2.1 MB/s 0:00:00
  ERROR: Cannot unpack file /tmp/pip-unpack-zm07m3n7/gval (downloaded from /tmp/pip-install-xelycx_f/gval_51a2513586ca49a88ef882afeaae6bd0, content-type: text/html; charset=utf-8); cannot detect archive format
ERROR: Cannot determine archive format of /tmp/pip-install-xelycx_f/gval_51a2513586ca49a88ef882afeaae6bd0
```

**Comment by fernando-aristizabal** [comment]
Hey Jason, Thanks for taking a look. Please try the following instead: 

`pip install 'gval @ git+https://github.com/NOAA-OWP/gval'`

**Comment by fernando-aristizabal** [comment]
@jarq6c This should be addressed by #105. Thanks @GregoryPetrochenkov-NOAA!

**Comment by jarq6c** [comment]
That solved the issue. I was able to install `gval`.

Thanks!
